package.json

```diff
@@ -461,6 +461,11 @@
         "command": "terraform.cloud.workspaces.refresh",
         "title": "Refresh",
         "icon": "$(refresh)"
+      },
+      {
+        "command": "terraform.cloud.workspaces.selectProject",
+        "title": "Select Project",
```

Comment:
I think 'Filter by Project' fits more to the concept here, as the expanded concept is 'Filter Workspaces by Project'. The expanded 'Select Project to Filter Workspaces' doesn't exactly roll off the tongue. :)

If we do agree to change the title, I think command should go along as well.